### DISCUSSION TITLE
[Roadmap] Skill tree reset / Expansion re-rolls

### DISCUSSION THREAD

**Discussion by future2back** [sunflower-land/sunflower-land#2187] [Ideas]
The roadmap for 2023 has just been announced, I believe the Skill tree reset and Expansion re-rolls changes could be game breaking.

Skill tree reset is a lack of respect for those who think in the long term. Can you guys list me some MMORPG where built reset was successful case?

Today without Expansion re-rolls its cheaper to me buy resources outside then make it on my farm, can you imagine what will happens with this change? I cant, but sure that will be not good.

These changes make it a Play to Win game. I believe you should rethink about these changes.

**Comment by InuBakaBo** [comment]
1. You do understand that some skills have no effect at all as the feature is not yet implemented like Plant Whisper and Stone monsters etc so unfair for those who put points in those
2. The skill descriptions are very vague, there's no details of what% rate things would trigger so again unfair for those who didn't know exactly how certain skills work
3. Skill tree is going to be restructured and revamped with different trees and skills
4. Buying resources outside the game is fine as it's part of the economy/market system mechanic. Hence why they teamed up with NiftySwap and even implemented the goblin balloon. Trading is part of the game
5. How is it a pay to win when everyone will have access to the skill reset feature. One free reset will likely be given to everyone for free when the new skill tree system comes out, and also it's another mechanic that burns resources. Additionally this promotes the community to discuss strategies, combinations and builds. Keeping the community engage keeps the game active and alive
